hq/README.md

```diff
@@ -29,32 +29,29 @@ N.B. If you are using macOS you may need to install `coreutils` before you can r
 ### Getting started
 
 ### AWS Configuration
-Go to AWS IAM console (security account) and clone `security-hq-dev` user to security-hq-<yourname>
-Create an Access key from the Security Credentials tab and save the access key under `security-test` section in your `~/.aws/credentials` file
-You should have an entry looks like below
+
+Using the Security Admin account, go to AWS IAM console and clone the `security-hq-dev` user to `security-hq-<yourname>`.
+
+At the last step of the creation of this new user, you will be presented with an *Access key ID* and a *Secret access key* that you are going to use during the next step. (Alternatively, you can create those credentials from an existing user from the Security Credentials tab.)
```

Comment:
Maybe remove:
`(Alternatively, you can create those credentials from an existing user from the Security Credentials tab.)`

I would rather encourage people to create a user specific to them and this use, than extend another or create multiple keys.

Reply:
Done!